lang-portal/frontend/.env.example

```
@@ -6,11 +6,27 @@ NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_...
+NEXT_PUBLIC_POSTHOG_HOST=
+NEXT_PUBLIC_POSTHOG_UI_HOST=
+POSTHOG_PROXY_PATH=
+POSTHOG_PROXY_TARGET=
+POSTHOG_PROXY_ASSETS_HOST=
 
 # Sentry Configuration
 SENTRY_ORG=sorami
 SENTRY_PROJECT=lang-portal-frontend
-SENTRY_DSN=https://5f2aa379610a248fdb8e476f9680476a@o4509562367705088.ingest.de.sentry.io/4509562384023632
\ No newline at end of file
+SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_DSN key should go before the SENTRY_ORG key

(UnorderedKey)